lib/bgs/student_school.rb

```diff
@@ -0,0 +1,142 @@
+# frozen_string_literal: true
+
+module BGS
+  class StudentSchool < Service
+    def initialize(proc_id:, vnp_participant_id:, payload:, user:)
+      @proc_id = proc_id
+      @vnp_participant_id = vnp_participant_id
+      @dependents_application = payload['dependents_application']
+
+      super(user)
+    end
+
+    def create
+      create_child_school
+      create_child_student
+    end
+
+    private
+
+    def create_child_school
+      with_multiple_attempts_enabled do
+        service.vnp_child_school.child_school_create(
+          child_school_params
+        )
+      end
+    end
+
+    def create_child_student
+      with_multiple_attempts_enabled do
+        service.vnp_child_student.child_student_create(
+          child_student_params
+        )
+      end
+    end
+
+    # rubocop:disable Metrics/AbcSize
+    # rubocop:disable Metrics/MethodLength
+    def child_school_params
```

Comment:
To me it seems like the parameter mapping doesn't belong in the service class. What about creating a model/models to clean this up a bit?

Reply:
@LindseySaari I wasn't _quite_ sure if this is what you were suggesting but I didn't want to come back empty handed. What do you think about this?

https://github.com/department-of-veterans-affairs/vets-api/pull/4546/files#diff-44bb8d03cb5942b139547c36a9525c68